static/scss/calculator.scss

```diff
@@ -81,3 +81,56 @@
     }
   }
 }
+
+@include breakpoint(phone) {
+  .financial-aid-calculator, .confirm-dialog {
```

Comment:
can you nest the mobile-specific code within the block for these selectors that we already have above?